go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/lima-vm/go-qcow2reader
 
 go 1.20
+
+// Testing requirements
+require github.com/lima-vm/go-qcow2reader/test v0.4.0
+
+replace github.com/lima-vm/go-qcow2reader/test v0.4.0 => ./test
```

Comment:
Why this has to be a separate module?

Reply:
It can be in the package, I tried to avoid building this as part of the go-qcow2reader module. Removing the extra module will make it simpler.

Reply:
I don't like maintaining multiple modules (with an exception for `cmd/go-qcow2reader-example`, as it was inevitable to isolate the thirdparty zstd dependency from the core module).

IIUC, just splitting the package is enough to omit the corresponding object code from the binary of Lima.
Even if it was not enough, the footprint of the object code would be negligible (just a few KiB?)

Reply:
The qemuimg package is tiny. I'll remove the module. Should we keep the test directory to make it more clear that this is a test only code?

Reply:
> Should we keep the test directory to make it more clear that this is a test only code?

Either would be fine

Reply:
Module removed.